### PR TITLE
Add PHPStan lvl 8, and fix some tests

### PR DESCRIPTION
**Pull request by rvanlaak**
Great bundle! Hope some minor support from my side helps :-) 

I've added PHPStan static analysis and started with fixing some of the 27 errors it reported on level 8.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,5 +36,5 @@ jobs:
     - name: Install dependencies
       run: composer install --prefer-dist --no-progress
 
-    - name: Run test suite
-      run: composer run-script test
+    - name: Run suite ${matrix.suite}
```
Comment (yceruto) [inline .github/workflows/ci.yml:39]: it should be `${{ matrix.suite }}` instead?

**Review by yceruto — COMMENTED** [review]
Hi @rvanlaak, I am glad to see that you are interested in this new bundle 🚀

File: composer.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
         "phpunit/phpunit": "^9.5",
         "symfony/form": "^6.2",
         "symfony/twig-bridge": "^6.2",
-        "twig/twig": "^3.5"
+        "twig/twig": "^3.5",
+        "phpstan/phpstan": "^1.9"
```
Comment (yceruto) [inline composer.json:26]: I was thinking of installing "vimeo/psalm" instead as that's the static tool used by the moneyphp library

**Comment by yceruto** [comment]
Closing in favor of https://github.com/yceruto/money-bundle/pull/3, thanks a lot @rvanlaak !

File: composer.json (hunk 2 above)
Comment (rvanlaak) [reply composer.json:26]: Fair point!

**Pull request closed without merging**